### ISSUE TITLE
Multiplication following a int8 transposed convolutions isn't constant folded

### ISSUE THREAD

**Issue by pkgoogle** [google-ai-edge/ai-edge-torch#393]
### Description of the bug:

Original Issue: https://github.com/tensorflow/tensorflow/issues/57680
Opening on behalf of @lgeiger

When converting a model that used int8 quantization aware training, conversion of transposed convolutions followed by a scalar multiplication fails.

The converter isn't able to correctly constant fold the per-tensor fake quantized weights and the scalar multiplication which is a common patter when using transposed convolutions followed by batch normalisation layers. This is a follow-up issue to #53766

### 1. System information

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): macOS / Ubuntu
- TensorFlow installation (pip package or built from source): pip
- TensorFlow library (version, if pip package or github SHA, if built from source): 2.10.0 / 2.11.0-dev20220913

### 2. Code

A minimal reproduction of the issue is available in [this notebook](https://colab.research.google.com/drive/1XM12NAzoiWRJS12uYyzNMsfi571fiauA?usp=sharing). Re-run the notebook to show netron visualisations showing the conversion problem.

**Comment by gaikwadrahul8** [comment]
This issue originally reported by @lgeiger has been moved to this dedicated repository for [ai-edge-torch](https://github.com/google-ai-edge/ai-edge-torch) to enhance issue tracking and prioritization. To ensure continuity, we have created this new issue on your behalf.

We appreciate your understanding and look forward to your continued involvement.

**Comment by aravindmurali22** [comment]
Hello @lgeiger,
Since this issue was opened, significant updates and improvements have been released that may have already addressed your concern. We are closing this issue to maintain a current and relevant backlog. Please test using the latest version of the library. If the issue persists, kindly open a new report with updated reproduction steps so we can investigate it within our current development workflow.
Thank you.
